Yosemite/Yosemite/Stores/ReceiptStore.swift

```diff
@@ -56,4 +62,67 @@ private extension ReceiptStore {
         let renderer = ReceiptRenderer(content: content)
         onContent(renderer.htmlContent())
     }
+
+    func loadReceipt(order: Order, onCompletion: @escaping (Result<CardPresentReceiptParameters, Error>) -> Void) {
+
+        guard let outputURL = try? fileURL(order: order),
+              FileManager.default.fileExists(atPath: outputURL.path) else {
+            let error = ReceiptStoreError.fileNotFound
+            onCompletion(.failure(error))
+            return
+        }
+
+        guard let receiptContent: ReceiptContent = try? fileStorage.data(for: outputURL) else {
+            DDLogError("⛔️ Unable to load receipt metadata for order: \(order.orderID)")
```

Comment:
Could we make this an info or at most a warn? Most orders won't have in-person payment receipts

Reply:
Sure, I'll make it a warning. This should only be reached when the file exists and we fail to parse it though.

Reply:
I got it on an older order:

`2021-05-12 10:04:13.483040-0700 WooCommerce[3172:2030342] ⛔️ Unable to load receipt metadata for order: 8295`